Exercise_1.py

```diff
@@ -5,7 +5,16 @@
 # if present, else returns -1 
 def binarySearch(arr, l, r, x): 
   
-  #write your code here
+    #write your code here
+    while l<r:
```

Comment:
The terminating condition should be "while l <= r", because for given array [10], you will not be able to find 10 since l and r are both 0 and your program will never enter the while loop.